content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -42,35 +42,71 @@ Istio control plane, Envoy can then communicate with the **single**
 Istio control plane and form a mesh network across multiple Kubernetes
 clusters.
 
-## Deploy all Kubernetes clusters to be used in the mesh
-
-After deployment of remote clusters, each one will have a
-credentials file associated with the admin context typically
-located in `$HOME/.kube/config`.
+## Create service account in remote clusters
+
+The Istio control plane requires access to all clusters in the mesh to
+discover services, endpoints, and pod attributes.  The following
+describes how to create a Kubernetes service account with the minimal
+RBAC access required.  The procedure then generates a `kubeconfig` file
+for the cluster using the credentials of the service account.
+
+Perform these steps for each remote cluster in the mesh:
+
+1. Create `ClusterRole`, `istio-reader`, for the Istio control plane access:
```

Comment:
I don't see particular harm in a followup PR to integrate this into the istio helm chart.